libs/ui/src/lib/top-holdings/top-holdings.component.html

```diff
@@ -57,8 +58,59 @@
       </td>
     </ng-container>
 
+    <ng-container matColumnDef="expandedDetail">
+      <td
+        *matCellDef="let element"
+        mat-cell
+        [attr.colspan]="displayedColumns.length"
+      >
+        <div
+          [@detailExpand]="element.expand ? 'expanded' : 'collapsed'"
+          class="allocationByParent"
+        >
+          <div class="holdingParents">
+            <mat-list>
```

Comment:
This is not really possible since it will now also include a symbol cell.
I just tried some various layouts with a table but I personally think that this will make the UI less intuitive as it looks too much the same.

See scrreenshot below for what I have so far:
![image](https://github.com/user-attachments/assets/fe729107-4556-4bb7-ac2a-dae4a6a61631)